src/services/tracking/abTesting.ts

```diff
@@ -1,7 +1,9 @@
 /**
  * Holds current A/B test identifiers.
  */
-export const enum AbTest {}
+export const enum AbTest {
+  HUMAN_DESCRIPTION = 'human_description',
```

Comment:
```suggestion
  HUMAN_DESCRIPTION = 'human-readable',
```
I would look for this in GA if I were looking for it in a month.